flocker/provision/_aws.py

```diff
@@ -51,6 +53,32 @@
 INSTANCE_TIMEOUT = 300
 
 
+class EliotLogHandler(logging.Handler):
+    # Whitelist ``"msg": "Params:`` field for logging.
+    _to_log = {"Params"}
```

Comment:
I looked at the `Params` from logged from a bunch of boto api calls and they don't seem that interesting....and often empty. Are you sure this is going to be helpful for debugging?

I notice that the filename and function name are also available in the record. Could those be used to filter out only the functions we're interested in?
 
 * s3.get_all_buckets
```
In [7]: s3.get_all_buckets()
{'threadName': 'MainThread', 'name': 'boto', 'thread': 140470047586048, 'created': 1454440854.63155, 'process': 28557, 'processName': 'MainProcess', 'args': (), 'module': 'connection', 'filename': 'connection.py', 'levelno': 10, 'exc_text': None, 'pathname': '/home/richard/.virtualenvs/4060/lib/python2.7/site-packages/boto/connection.py', 'lineno': 903, 'msg': 'Params: {}', 'exc_info': None, 'funcName': '_mexe', 'relativeCreated': 42857.568979263306, 'levelname': 'DEBUG', 'msecs': 631.5500736236572}
> /home/richard/projects/HybridLogic/flocker/flocker/provision/_aws.py(67)emit()
-> Message.new(
(Pdb) c
{'threadName': 'MainThread', 'name': 'boto', 'thread': 140470047586048, 'created': 1454440854.63155, 'process': 28557, 'processName': 'MainProcess', 'args': (), 'module': 'connection', 'filename': 'connection.py', 'levelno': 10, 'exc_text': None, 'pathname': '/home/richard/.virtualenvs/4060/lib/python2.7/site-packages/boto/connection.py', 'lineno': 903, 'msg': 'Params: {}', 'exc_info': None, 'funcName': '_mexe', 'relativeCreated': 42857.568979263306, 'levelname': 'DEBUG', 'msecs': 631.5500736236572}
> /home/richard/projects/HybridLogic/flocker/flocker/provision/_aws.py(67)emit()
-> Message.new(

```

* ec2.get_all_volumes

```
In [9]: ec2.get_all_volumes()
{'threadName': 'MainThread', 'name': 'boto', 'thread': 140470047586048, 'created': 1454440935.590707, 'process': 28557, 'processName': 'MainProcess', 'args': (), 'module': 'connection', 'filename': 'connection.py', 'levelno': 10, 'exc_text': None, 'pathname': '/home/richard/.virtualenvs/4060/lib/python2.7/site-packages/boto/connection.py', 'lineno': 903, 'msg': "Params: {'Action': 'DescribeVolumes', 'Version': '2014-10-01'}", 'exc_info': None, 'funcName': '_mexe', 'relativeCreated': 123816.72596931458, 'levelname': 'DEBUG', 'msecs': 590.7070636749268}
> /home/richard/projects/HybridLogic/flocker/flocker/provision/_aws.py(67)emit()
-> Message.new(
(Pdb) c
{'threadName': 'MainThread', 'name': 'boto', 'thread': 140470047586048, 'created': 1454440935.590707, 'process': 28557, 'processName': 'MainProcess', 'args': (), 'module': 'connection', 'filename': 'connection.py', 'levelno': 10, 'exc_text': None, 'pathname': '/home/richard/.virtualenvs/4060/lib/python2.7/site-packages/boto/connection.py', 'lineno': 903, 'msg': "Params: {'Action': 'DescribeVolumes', 'Version': '2014-10-01'}", 'exc_info': None, 'funcName': '_mexe', 'relativeCreated': 123816.72596931458, 'levelname': 'DEBUG', 'msecs': 590.7070636749268}
> /home/richard/projects/HybridLogic/flocker/flocker/provision/_aws.py(67)emit()
-> Message.new(
(Pdb) c

```

* ec2.get_all_instances

```
In [10]: ec2.get_all_instances()
{'threadName': 'MainThread', 'name': 'boto', 'thread': 140470047586048, 'created': 1454440961.704781, 'process': 28557, 'processName': 'MainProcess', 'args': (), 'module': 'connection', 'filename': 'connection.py', 'levelno': 10, 'exc_text': None, 'pathname': '/home/richard/.virtualenvs/4060/lib/python2.7/site-packages/boto/connection.py', 'lineno': 903, 'msg': "Params: {'Action': 'DescribeInstances', 'Version': '2014-10-01'}", 'exc_info': None, 'funcName': '_mexe', 'relativeCreated': 149930.7999610901, 'levelname': 'DEBUG', 'msecs': 704.7810554504395}
> /home/richard/projects/HybridLogic/flocker/flocker/provision/_aws.py(67)emit()
-> Message.new(
(Pdb) c
{'threadName': 'MainThread', 'name': 'boto', 'thread': 140470047586048, 'created': 1454440961.704781, 'process': 28557, 'processName': 'MainProcess', 'args': (), 'module': 'connection', 'filename': 'connection.py', 'levelno': 10, 'exc_text': None, 'pathname': '/home/richard/.virtualenvs/4060/lib/python2.7/site-packages/boto/connection.py', 'lineno': 903, 'msg': "Params: {'Action': 'DescribeInstances', 'Version': '2014-10-01'}", 'exc_info': None, 'funcName': '_mexe', 'relativeCreated': 149930.7999610901, 'levelname': 'DEBUG', 'msecs': 704.7810554504395}
> /home/richard/projects/HybridLogic/flocker/flocker/provision/_aws.py(67)emit()
-> Message.new(
(Pdb) c

```